commercetools.Sdk/Tests/commercetools.Sdk.Domain.Tests/MoneyTest.cs

```diff
@@ -8,7 +8,7 @@ public class MoneyTest
         [Fact]
         public void MoneyToDecimal()
         {
-            var m = new Money() { CurrencyCode = "EUR", CentAmount = 123456 };
+            var m = new Money() { CurrencyCode = "EUR", CentAmount = 123456, FractionDigits = 2};
```

Comment:
with a digit fallback in the calculation this adjustment should not be necessary